manifest-list.md

```diff
@@ -14,9 +15,8 @@ This section defines the `application/vnd.oci.image.manifest.list.v1+json` [medi
 
 - **`mediaType`** *string*
 
-  This REQUIRED property contains the media type of the manifest list.
-  For this version of the specification, this MUST be set to `application/vnd.oci.image.manifest.list.v1+json`.
-  For the media type(s) that this is compatible with, see the [matrix](media-types.md#compatibility-matrix).
+  This property is *reserved* for use, to [maintain compatibility][matrix].
+  When used, this field contains the media type of this document, which differs from the [descriptor](descriptor.md#properties) use of `mediaType`.
```

Comment:
I kinda agree, this confused me on my read-through. Can we just say something like: 
```
This OPTIONAL property is RESERVED by the specification for backwards compatibility with older versions of Docker. The value of this field will not change. This field MAY be removed in a future version of the specification.
```
and leave it at that.